src/CodeSigning/CodeSigning/Commands/GetAzureCodeSigningCustomerEku.cs

```diff
@@ -0,0 +1,101 @@
+using System.Security;
+
+namespace Microsoft.Azure.Commands.CodeSigning
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzurePrefix + "CodeSigningCustomerEku", DefaultParameterSetName = ByAccountProfileNameParameterSet)]
+    [OutputType(typeof(string))]
+    public class GetAzureCodeSigningCustomerEku : CodeSigningCmdletBase
+    {
+        #region Parameter Set Names
+
+        private const string ByAccountProfileNameParameterSet = "ByAccountProfileNameParameterSet";
+        private const string ByMetadataFileParameterSet = "ByMetadataFileParameterSet";
+
+        #endregion
+
+        #region Input Parameter Definitions
+
+        /// <summary>
+        /// Account Profile name
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = ByAccountProfileNameParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The account name of Azure CodeSigning.")]       
+        [ValidateNotNullOrEmpty]
+        public string AccountName { get; set; }
+
+        [Parameter(Mandatory = true,
+            Position = 1,
+            ParameterSetName = ByAccountProfileNameParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The certificate profile name of Azure CodeSigning account.")]        
+        [ValidateNotNullOrEmpty()]
+        public string ProfileName { get; set; }
+        [Parameter(Mandatory = true,
+            Position = 2,
+            ParameterSetName = ByAccountProfileNameParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The endpoint url used to submit request to Azure CodeSigning.")]
+        public string EndpointUrl { get; set; }
+
+
+        /// <summary>
+        /// Metadata File Path
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = ByMetadataFileParameterSet,
+             ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Metadata File path. Cmdlet constructs the FQDN of an account profile based on the Metadata File and currently selected environment.")]        
+        [ValidateNotNullOrEmpty]
+        public string MetadataFilePath { get; set; }
+               
+        #endregion
+
+        public override void ExecuteCmdlet()
+        {
+            string eku;
+
+            if (!string.IsNullOrEmpty(AccountName))
+            {
+                eku = CodeSigningServiceClient.GetCodeSigningEku(AccountName, ProfileName, EndpointUrl);
+                WriteEku(eku);
+            }          
+            else if (!string.IsNullOrEmpty(MetadataFilePath))
+            { 
+                eku = CodeSigningServiceClient.GetCodeSigningEku(MetadataFilePath);
+                WriteEku(eku);            
+            }
+        }
+
+        private void WriteEku(string eku)
+        {  
+            WriteObject(eku.Split(','));         
```

Comment:
Please remove the spaces at the end of line.

Reply:
Done.